data-ingestion-apis/opentelemetry-metrics.mdx

```diff
@@ -42,56 +42,130 @@ curl -X POST "https://demo.middleware.io/v1/metrics" \
 -H "Content-type: application.json" \
 -d @ << EOF
 {
-    "resource_metrics": [
-        {
-            "resource": {
-                "attributes": [
-                    {
-                        "key": "mw.account_key",
-                        "value": {
-                            "string_value": "xxxxxxxxxx"
-                        }
-                    },
-                    {
-                        "key": "mw.resource_type",
-                        "value": {
-                            "string_value": "custom"
-                        }
+  "resourceMetrics": [
```

Comment:
Did this change recently? It was resource_metrics

Reply:
Not sure. @eduardo-mw can you confirm?

Reply:
Adding @bhogayatakb to this conversation to double check

Reply:
This should be `resource_metrics`, See Reference [here](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L44)

Reply:
@lukebarrett24  Once that change is made I can approve asap